From d69426da429064697dc66d6ab7d08cbcc09031bd Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Sun, 2 Jun 2019 02:50:21 +0200
Subject: [PATCH] modules: update packages

5d3cb594e Merge pull request #9058 from micmac1/sqlite3-arm-endian-owrt18
d13f831ba sqlite3: fix arm endian issue
48687a581 Merge pull request #8966 from BKPepe/18.06-ytdl
5387239ad youtube-dl: update to version 2019.05.11
76ebe2be0 Merge pull request #8765 from BKPepe/1806-urllib3
ec5e894ce Merge pull request #8954 from jonathanunderwood/openwrt-18.06-getdns-stubby-from-master
54928eb69 nfs-kernel-server: use portmap, drop rpcbind
09e18eef7 stubby: set PKG_SOURCE_VERSION using PKG_VERSION
f8c8b96e7 stubby: remove PKG_SOURCE from Makefile
28c328d66 stubby: add support for TLS configuration options
eec23a91b stubby: add support for tls_port resolver config option (#8889)
5415cc7b2 stubby: update to 0.2.6
b1cc26adf getdns: update to version 1.5.2
813d49c69 stubby: Update PKG_RELEASE
3b498844a stubby: fix handling of pkis in config (#8888)
95daecd81 stubby: fixed duplicate IPv6 address in stubby.yml; fixed typo in README.md
eca82ce3d Merge pull request #8938 from lucize/doveup18
2c87aeba1 dovecot: update to 2.2.36.1
8459433eb python-urllib3: update to 1.24.3
014106741 Merge pull request #8781 from neheb/pros
41c85f8ef Merge pull request #8767 from neheb/cr2
9e4966bd3 Merge pull request #8816 from neheb/lo
55b505f82 Merge pull request #8815 from neheb/gr
1663a0186 Merge pull request #8881 from micmac1/maria_10.1.39
78328ebe3 libxslt: add PKG_CPE_ID for proper CVE tracking
816c7aed6 Merge pull request #8882 from micmac1/xslt-cve-18
2eb2a94da mariadb: bump to 10.1.39
d03db02bc libxslt: backport patch for CVE-2019-11068
d3c324d8b Merge pull request #8871 from neheb/ad
fb9238b61 addrwatch: Add -std=gnu89 to fix compilation
5fae371ab Merge pull request #8779 from neheb/libsec
3ca88ff6d Merge pull request #8780 from neheb/apc
1d44e408f Merge pull request #8770 from neheb/es
759e83674 Merge pull request #8772 from neheb/oa
ea1429f69 Merge pull request #8778 from neheb/liba
46d30c221 Merge pull request #8850 from EricLuehrsen/unbound_dhcp_1806
7f7fda67b add rpcbind
80429e0a5 unbound: fix local dns synthesis corner cases
1f77a0582 Merge pull request #8777 from neheb/sendm
071a926dd Merge pull request #8817 from neheb/bonnnnn
ddc48c3d5 bonnie++: Remove wrongly added patch during cherry-pick
21de3c1ac loudmouth: Disable debug and fix compilation
79e69bf41 grilo: Remove build dependency on perl-xml-parser
c3f6660b8 nfs-kernel-server: fix missing libbsd dependency
d2f254a72 nfs-kernel-server: update to 2.3.3
dac119d49 nfs-kernel-server: Switch to xz tarball
f1ab32ab6 nfs-kernel-server: switch to libtirpc, enable ipv6
5ab569a06 nfs-kernel-server: fix missing host symbol res_querydomain/missing-include-dir
20809ba6a nfs-kernel-server: add -Wno-error=format-security to TARGET_CFLAGS to a fix compily error
2c137a7a5 nfs-kernel-server: update to 2.3.2
765ae2798 libtirpc: enable host build
9b3414fd1 znc: Clean up Makefile and fix compilation on PPC
a8316a5c6 nano: update to 4.2
f69a00efb Merge pull request #8764 from BKPepe/1806-youtubedl
024543fb1 Merge pull request #8766 from neheb/qu
c69c1aea4 Merge pull request #8768 from neheb/cmd
d018e0946 Merge pull request #8776 from neheb/luas
9f74ef0f8 luasec: cleanup Makefile
bbfd71a59 luasec: use gcc to link instead of ld
7059f9476 luasec: Replace -fPIC with $(FPIC)
a3d99ad90 youtube-dl: update to version 2019.04.24
9e532f225 Merge pull request #8773 from neheb/cla
bcf889b36 intltool: Add patch to fix regular expressions and avoid warnings in perl 5.22 and later.
2653eabd4 prosody: Change linker from LD to CC
c15b779b8 prosody: Update prosody to 0.11.2 Update Fixes CVE-2018-10847
9104a1f14 apcupsd: preset shutdown binary
e484057d0 libseccomp: Update to 2.3.3
f61ee11d0 libartnet: Fix compile with GCC7
9d2bf88ec sendmail: link with gcc instead of ld
dd2c1e03e Merge pull request #8769 from neheb/da
3a478756f Merge pull request #8774 from neheb/psq
cf4537574 psqlodbc: Update to 11.00.0000
8ca29a4f3 psqlodbc: bump to 10.03.0000, fix --with-unixodbc
81a6687a6 Merge pull request #8771 from neheb/pia
fa060f40e clamsmtp: Really fix compilation
850126c0d oath-toolkit: Add patch from Fedora to fix compilation
fd52a3df0 pianod: Update to 174.09 release (removes BSD dependencies)
a0ca3e308 espeak: Fix build issues. Cleanup patches
d02a17b35 darkstat: Remove libbsd dependency
0a4f26f3c cmdpad: Fix compile
27af84f57 crtmpserver: Update to official git repo
1a7d79d98 quassel-irc: Update to latest version
873762ce2 quasselc: Update to latest commit
c8c8c1a4a Merge pull request #8759 from neheb/bon2
76132b044 bonnie++: Update to 1.98
df7f21bcd Merge pull request #8718 from neheb/lcd
588f971f0 Merge pull request #8716 from neheb/wifi
4e0a836c5 Merge pull request #8717 from neheb/setser
403d40d91 Merge pull request #8720 from neheb/iio
03a4afe5f Merge pull request #8719 from neheb/unix
d42547cd3 Merge pull request #8733 from jefferyto/openwrt-18.06-python-cve-2019-9636
fbcac2d53 python,python3: Fix CVE-2019-9636 - urlsplit missing NFKC normalization
01f90caa1 Merge pull request #8706 from nxhack/18_06_icu_support_reiwa
e0bee26f4 libiio: add missing dependency to zlib
7220dab75 unixodbc: save unixodbc_conf.h for host build
01a2f4353 unixodbc: fix clean-build compilation
bd380a3c7 unixodbc: Fix LIB_PREFIX in host build
9b5b23f44 lcd4linux: Update to 1204
c26a03a1d setserial: Don't build docs to remove nroff dependency
2a85006ab wifidog: Change to use TLS above 1.0
94b07876e icu: [openwrt-18.06] support for new Japanese era Reiwa
c9aedd9a5 nano: update to 4.1
49e759b28 Merge pull request #8638 from mwarning/zerotier
890993d40 Merge pull request #8477 from BKPepe/openwrt-18.06_perl
d6ed5c22b net/mosquitto: correct config file option name
05e16dd22 zerotier: fix multiple instance handling and port setting
59c6457ec zerotier: update to version 1.2.12
4db102755 Merge pull request #8595 from EricLuehrsen/openwrt-18.06
43c1d6c2a unbound: correct forward of root domain
b8970833b perlmod: fix ability to build module out-of-feed

(cherry picked from commit bbd52db5111307a3e4d0532b2c953a2ba5dcb440)
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 3ed0e6f2..504ac323 100644
--- a/modules
+++ b/modules
@@ -6,7 +6,7 @@ OPENWRT_COMMIT=67619619195aa57b2d3169b659e677cd8b879e2a
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-18.06
-PACKAGES_PACKAGES_COMMIT=b7bcf24a00153f0c99ff2e01fa33876680c6128d
+PACKAGES_PACKAGES_COMMIT=5d3cb594e49132158ca7b41f4246421078e8f92a
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt-routing/packages.git
 PACKAGES_ROUTING_BRANCH=openwrt-18.06
-- 
2.17.1

